### ISSUE TITLE
subctl join’s --globalnet flag no longer seems useful

### ISSUE THREAD

**Issue by skitt** [submariner-io/subctl#43]

**Comment by sridhargaddam** [comment]
> @sridhargaddam can keep me honest here, but IIRC this was added based on user's feedback.This is a very niche use-case where in one central/hub cluster we are disabling Globalnet, however all the member clusters that are joined to the hub cluster use Globalnet.

Yes that's correct @nyechiel. Although we do not recommend using a mixed deployment, if configured properly this flag allows Submariner to support some niche use-cases like centralised storage which needs access from tenant clusters. We should get some feedback from the team which requested this use-case before deprecating this flag.
